shoop/core/models/_payments.py

```diff
@@ -34,3 +34,7 @@ class Payment(MoneyPropped, models.Model):
     class Meta:
         verbose_name = _('payment')
         verbose_name_plural = _('payments')
+
+    @property
+    def currency(self):
+        return self.amount.currency
```

Comment:
What's the point of this? Why can't the API user just use `payment.amount.currency` (or `payment.foreign_amount.currency`)?